src/main/java/org/cadixdev/mercury/mixin/MixinRemapperVisitor.java

```diff
@@ -364,4 +365,59 @@ private static MethodSignature convertSignature(final String name, final IMethod
         return new MethodSignature(name, new MethodDescriptor(parameters, convertType(binding.getReturnType())));
     }
 
+    private void remapPrivateMixinTarget(
+        final AST ast, final TypeDeclaration typeDeclaration, final ITypeBinding typeBinding
+    ) {
+        IAnnotationBinding[] annotations = typeBinding.getAnnotations();
+        List modifiers = typeDeclaration.modifiers();
+        for (Object modifier : modifiers) {
+            if (modifier instanceof NormalAnnotation) {
+                NormalAnnotation normalAnnotation = (NormalAnnotation) modifier;
+                for (Object raw : normalAnnotation.values()) {
+                    MemberValuePair pair = (MemberValuePair) raw;
+                    if ("targets".equals(pair.getName().getIdentifier())) {
+                        Expression targets = pair.getValue();
+                        if(targets instanceof StringLiteral){
+                            remapPrivateMixinTargetLiteral(ast, ((StringLiteral) targets));
+                        }
+                        else if (targets instanceof ArrayInitializer) {
+                            List expressions = ((ArrayInitializer) targets).expressions();
+                            for (Object expression : expressions) {
+                                if (expression instanceof StringLiteral) {
+                                    remapPrivateMixinTargetLiteral(ast, (StringLiteral) expression);
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    private void remapPrivateMixinTargetLiteral(AST ast, StringLiteral literal) {
+        String className = literal.getLiteralValue().replace('.', '/');
+        if (!className.isEmpty()) {
+            ClassMapping<?, ?> classMapping = mappings.computeClassMapping(className)
```

Comment:
I'm currently testing this with remapping `net/minecraft/network/NetworkManager$2` to `el` and changing `computeClassMapping` to `getTopLevelClassMapping` _actually for sure_ worked here. `computeClassMapping` leaves it as `ej$2`.